### PR TITLE
Improve baseline alignment of form elements

### LINKED ISSUE #361: Label, input value and appended button vertical alignment issue
See screenshot 1, appended button text looks to be out of line, although it's actually the input value (see screenshot2). Had a quick attempt at a fix but button is too large (see screenshot 3).

Needs a deeper look.

### PR DESCRIPTION
**Pull request by Stanton**
Closes #361

### CODE REVIEW

**Comment by Stanton** [comment]
Holding this PR until James returns in the New Year.

File: stylesheets/_component.forms.scss [hunk 1]
```diff
@@ -170,9 +170,21 @@ fieldset {
     }
 }
 
-.legend {
-    font-size: $font-size-large;
-    margin-bottom: .5em;
+fieldset {
```
Comment (jamesjacobs) [inline stylesheets/_component.forms.scss]: Feels like we should keep the removed `>` here. I can see some potential issues around nested fieldsets (dob fields)

Comment (Stanton) [inline stylesheets/_component.forms.scss]: You might need to clarify this for me, I can't see a removed `>` in the diff?

**Comment by jamesjacobs** [comment]
Can you update this with develop please and resolve any conflicts? Tried testing in CXM but there's a lot missing by the looks of it.

**Pull request closed without merging**